### DISCUSSION TITLE
2007/10/19/%E7%94%A8%E6%88%B7%E7%95%8C%E9%9D%A2%E8%AE%BE%E8%AE%A1

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#185] [Announcements]
# 2007/10/19/%E7%94%A8%E6%88%B7%E7%95%8C%E9%9D%A2%E8%AE%BE%E8%AE%A1

新版本请查看： https://lv.qizhen.xyz

https://blog.qizhen.xyz/2007/10/19/%E7%94%A8%E6%88%B7%E7%95%8C%E9%9D%A2%E8%AE%BE%E8%AE%A1

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2009年12月20日 21:44 编辑
工具条可以自己做，无非就是把一些按钮摆放在一起。但是LabVIEW不适合做包含子窗口的界面。如果你需要多个界面切换，可以使用Tab控件。

回复
minsz
2009年12月19日 19:20 编辑
请问大侠,界面能否设计成一般软件那种上有各种工具及模块按钮,可以多页面显示,或者可多显示能组合.我现有个项目,一方面是测试界面模块,另外我要做仿真分析模块.谢谢

回复
Qizhen
2007年10月25日 08:44 编辑
详细内容正在写

回复
林
2007年10月23日 12:08 编辑
高手能否讲的再详细些,我想在我的程序中实现windows菜单风格的界面应该用到哪些控件呢?

回复
杰
2007年10月22日 19:30 编辑
其实做界面真的是一个问题。
虽然大多数时候，我基本上考虑的很少。

回复
BEcky
2007年10月20日 22:10 编辑
请问你学什么专业的？交流下～

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2008年07月8日 09:36 编辑
感谢你的补充。等用空再把这篇内容完善一下，把你提及的也加进来。谢谢！

回复
agilentvee
2008年07月8日 09:05 编辑
楼主说的不全,小弟补充几点:
在实际应用中的限制是经常变动的,比如序列号规则的改变.所以这时候使用XCONTROL比较方便.同时楼主提到的enum&ring最好先定义成自定义控件.
防止误操作个人认为最好方法就是在操作一个control同时disable其余所有的control

回复
BO
2008年07月7日 22:40 编辑
你好，无意间发现你的blog，有出书的打算？太好了，我现在国外读博，也发现国内的好的lv的教程太少了，期望书的出版。关注中。

回复
